demos/datepicker/icon-trigger.html

```diff
@@ -15,13 +15,18 @@
 				open: function() {
 					allowOpen = false;
 				}
-			});
+			} ),
+			widget = datepicker.datepicker( "widget" );
 
-		$( "<img src='images/calendar.gif' alt='Open Datepicker'>")
+		$( "<a href='#'>")
```

Comment:
Jörn noticed this should be usable with keyboard only, thats why I added a link. Switching to a button seems like a good solution.